### PR TITLE
[TW-906] API Governance and Security rule editor updates

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated "Adding custom rule" instructions for API Governance and Security pages
* Updated screenshots in the API Governance and Security pages
* Mentioned the new boilerplate rule, snippets, and autocomplete suggestions

### CODE REVIEW

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 1]
```diff
@@ -42,15 +42,30 @@ Once you import new rules from the library, you need to turn them on in the Gove
 
 > Only Team Admins with a [Developer role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can create custom API Governance rules.
 
-You can create new custom governance rules for Postman to evaluate your API's definition.
+You can create new custom governance rules for Postman to evaluate your API's definition. Postman provides you with a boilerplate rule to help you start writing your custom governance rules. You can also use snippets of commonly-used property-value pairs to help you write your custom governance rules.
+
+> You can write and add custom functions to your custom rules. For more information, see [Configuring custom functions](/docs/api-governance/configurable-rules/configuring-custom-functions/).
```
Comment (mark-anthony-postman) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:47]: Link not working - not sure what section this is supposed to point to

**Review by mark-anthony-postman — APPROVED** [review]
Looks great! Just one link to check 😀

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:47]: Thanks for catching this! This link points to a page that I'm working on in branch `tw-907-governance-custom-functions`.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 2]
```diff
@@ -42,15 +42,28 @@ Once you import new rules from the library, you need to turn them on in the Gove
...
+
+    > Postman will prompt you with suggestions as you enter text. Select one to autocomplete your rule.
+
 1. The rule must be valid YAML or JSON. Use the dropdown list to choose the correct syntax.
-1. Select OpenAPI 2.0 or OpenAPI 3.0 to choose which specification your new rule is valid for.
 1. Select **Create**.
-    <img alt="Create a custom API Governance rule" src="https://assets.postman.com/postman-docs/v10/api-governance-create-custom-rule-v10.jpg" width="800px"/>
-1. Find your new rule under **Custom Rules** and turn it on.
+
+    <img alt="Create a custom API Governance rule" src="https://assets.postman.com/postman-docs/v10/api-governance-create-custom-rule-v10-2.jpg" width="800px"/>
```
Comment (claireriley) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:64]: Since the text is a bit hard to read here, I'd recommend removing the width value so users can select it to open up the larger viewing pop-up
```suggestion
    <img alt="Create a custom API Governance rule" src="https://assets.postman.com/postman-docs/v10/api-governance-create-custom-rule-v10-2.jpg"/>
```

File: src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md [hunk 3]
```diff
@@ -51,16 +51,23 @@ Your team can turn individual security rules on or off to meet your development
...
 1. Define the rule in the editor. It must adhere to [custom rule guidelines](/docs/api-governance/configurable-rules/spectral/).
+
+    You can use a curated list of commonly-used property-value pair snippets to write your rules. Snippets are available in the right pane of the editor. Selecting a snippet adds the property-value pair automatically to your rule, helping you get started quickly with writing rules. Once added to your rule, you can edit the snippets to meet your specific requirements.
+
+    > Postman will prompt you with suggestions as you enter text. Select one to autocomplete your rule.
+
 1. The rule must be valid YAML or JSON. Use the dropdown list to choose the correct syntax.
-1. Select OpenAPI 2.0 or OpenAPI 3.0 to choose which specification your new rule is valid for.
 1. Select **Create**.
     <img alt="Create a custom API Security rule" src="https://assets.postman.com/postman-docs/v10/api-security-create-new-custom-rule-v10.jpg" width="800px"/>
```
Comment (claireriley) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-security-rules.md:69]: To allow users to select to open the larger view
```suggestion
    <img alt="Create a custom API Security rule" src="https://assets.postman.com/postman-docs/v10/api-security-create-new-custom-rule-v10.jpg"/>
```

**Review by claireriley — APPROVED** [review]
LGTM! 🌟

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 4]
```diff
@@ -42,15 +42,28 @@ Once you import new rules from the library, you need to turn them on in the Gove
 
 > Only Team Admins with a [Developer role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) can create custom API Governance rules.
```
Comment (savageCoder55) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:43]: Let's rephrase this to say that "Only members with an API Governance Manager role can..."

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md [hunk 5]
```diff
@@ -42,15 +42,28 @@ Once you import new rules from the library, you need to turn them on in the Gove
...
+
 1. The rule must be valid YAML or JSON. Use the dropdown list to choose the correct syntax.
-1. Select OpenAPI 2.0 or OpenAPI 3.0 to choose which specification your new rule is valid for.
 1. Select **Create**.
-    <img alt="Create a custom API Governance rule" src="https://assets.postman.com/postman-docs/v10/api-governance-create-custom-rule-v10.jpg" width="800px"/>
-1. Find your new rule under **Custom Rules** and turn it on.
+
+    <img alt="Create a custom API Governance rule" src="https://assets.postman.com/postman-docs/v10/api-governance-create-custom-rule-v10-2.jpg"/>
+
+1. Find your new rule under the **Rules** tab and turn it on.
```
Comment (savageCoder55) [inline src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:66]: Should we add the fact that this is available in the "Created by you" section.
Also we shouldn't say turn it on - rather say that add it to the workspace group on which you want the respective rule enforced.

**Review by savageCoder55 — CHANGES_REQUESTED** [review]
Couple comments added. Should be easy to fix.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md]: This is now updated in both the API Governance and API Security pages. For the API Security page, I wrote "API Security Manager" instead.

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 5 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md]: This is now updated in both the API Governance and API Security pages.

**Review by savageCoder55 — APPROVED** [review]

File: src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/api-governance/configurable-rules/configuring-api-governance-rules.md:43]: It's my understanding that we're no longer releasing the API Security Manager and API Governance manager team-level roles. I reverted the content back to its previous version.

**Review by savageCoder55 — APPROVED** [review]
